must-gather/gather

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+# copyright 2023.
+#
+# licensed under the apache license, version 2.0 (the "license");
+# you may not use this file except in compliance with the license.
+# you may obtain a copy of the license at
+#
+#     http://www.apache.org/licenses/license-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the license is distributed on an "as is" basis,
+# without warranties or conditions of any kind, either express or implied.
+# see the license for the specific language governing permissions and
+# limitations under the license.
+#
+
+set -eu -o pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+source "${SCRIPT_DIR}"/common
+
+print_usage() {
+    echo -e "Usage: /usr/bin/gather <gather-collection-path>"
+}
+
+get_kepler_instance() {
```

Comment:
I think we should get all `keplers` instead of just the `one` until we have a webhook that prevents creating any other kepler.

Reply:
any kepler not named `kepler` is ignored by operator.  
but changed script to capture all keplers

Reply:
Actually, it doesn't get ignored, the operator does run a reconcile and set the status to invalid.